### DISCUSSION TITLE
Regarding building individual apps using monorepo

### DISCUSSION THREAD

**Discussion by saleem-unifycare** [community/community#37316] [Actions]
HI,
 In my organization we are using monorepo which has different apps. We want to build and deploy to GKE only individual apps that has new code push unlike all apps.
we are using nodejs,reactjs,nx workspace
Kindly guide me on using github actions.

**Comment by ossanna16** [comment]
Hi there @saleem-unifycare and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by airtower-luna** [comment]
You can include a path filter for `push`, `pull_request`, or `pull_request_target` events you use, see: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore

E.g. you can provide a list of paths so the event will only trigger if any of those changed.
